docs/api-docs/getting-started/api-rate-limits.mdx

```diff
@@ -0,0 +1,120 @@
+
+```http filename="Standard plan example: Rate limit headers"
+X-Rate-Limit-Time-Window-Ms: 30000
+X-Rate-Limit-Time-Reset-Ms: 15000
+X-Rate-Limit-Requests-Quota: 150
+X-Rate-Limit-Requests-Left: 35
+```
+
+|Name | Description |
+|:------|:-----|
+| X-Rate-Limit-Time-Window-Ms | Shows the size of your current rate limiting window. In this case, it’s 5000 milliseconds. |
+| X-Rate-Limit-Time-Reset-Ms | Shows the size of your current rate limiting window. In this case, it’s 5000 milliseconds. |
+| X-Rate-Limit-Requests-Quota | Shows how many API requests are allowed in the current window for your client. In this case, 25 requests.|
+| X-Rate-Limit-Requests-Left | Details how many remaining requests your client can make in the current window before being rate limited.|
+
+Not seeing an HTTP header in your response? Continue reading on to the next section. 
+
+#### Seeing a `429` response?
+
+If you encounter a `429` response without an API HTTP header, this is an indication of the platform undergoing a load balance to prevent any downtime from happening.
+
+Here's an example of what a 429 response looks like:
+
+```http filename="Example: 429 response" showLineNumbers copy
+ HTTP/1.1 429 Too Many Requests
+ Date: Mon, 03 Feb 2022 20:36:00 GMT
+ Content-Type: application/json
+ X-Rate-Limit-Time-Reset-Ms: 15000
+```
+Continue to reduce the risk of an app outage by slowing down the amount of calls being made. This will reduce any strain on performance.  For more details on response headers, see [About Our API](/docs/start/about#bigcommerce-specific-response-headers).
+
+### Distribute the load
+
+To avoid spikes in API requests and ensure efficient use of your quota, spread out your API calls over time. The practice helps prevent hitting rate limits by balancing the number of requests sent in a given time frame.
+
+You can use strategies such as using asynchronous processes, batching requests, and using a queue. They will help you distribute the load evenly. You can also keep a smooth request rate by adjusting the rate based on the API response headers.
+
+### Caching results
+
+Cache frequent results to reduce API calls.
+
+### Retry requests responsibly
+
+To retry requests responsibly when using the BigCommerce API, follow these steps:
+
+1. Monitor for `429` Responses: When you receive a `429: Too Many Requests` response, it indicates that you've hit the rate limit.
+
+2. Check the Retry-After Header: Use the `X-Rate-Limit-Time-Reset-Ms` header to determine how long to wait before retrying. This header provides the time in milliseconds until your quota resets.
+
+3. Pause Before Retrying: Implement a delay in your code based on the `X-Rate-Limit-Time-Reset-Ms` value. For example, in PHP:
+
+```php filename="PHP" showLineNumbers copy
+ $milliseconds = $response->getHeader("X-Rate-Limit-Time-Reset-Ms");
+ usleep($milliseconds * 1000);
+```
+
```

Comment:
This is a good place to move content from line 70:
https://github.com/bigcommerce/docs/pull/325/files#diff-d2f1fa5aa0ab0a7a1e7fb819fd7c76acaa87881036fc61bf8a77eaf422dbbbd3R70

Reply:
I moved the content over so that it improves the flow. 👍